### ISSUE TITLE
Deploy bitwarden on QNAP NAS

### ISSUE THREAD

**Issue by ChriKn** [bitwarden/core#115]
Hi everybody,
Long-time KeePass user here, I wanted some change and a bit more ease of use (as well as compatibility for Firefox Quantum) bitwarden seems to be the perfect solution.
Only problem : I don't like relying on other servers and was thinking about self-hosting on my little QNAP NAS (TS-228 1 GB of RAM and Arm V7 1,1 Ghz Dual-Core)

My main goal is to get the NAS working as a home drive, Nextcloud instance working on it and also the server-side of bitwarden (if at all possible given the limited ressources)
From what I could get, the only way I have is to use Docker for bitwarden, but a minimum of 4GB of ram seems a bit extreme... 
Is it planned to make an official bitwaden qpkg to just get it running as an apllication in the background ? 
Is there maybe another solution I didn't see to make this work ?

TL;DR : Is it or will it be possible to install bitwarden easily (or with instructions) on a NAS with limited ressources ?

Thanks in advance !

**Comment by ChriKn** [comment]
Sad to read... This will probably keep me from using this great piece of software.
Would love to join you one day :)

**Comment by Nekotje** [comment]
i was also looking for something like this. Sad it isnt on the todo list.

**Comment by biotecs** [comment]
@ChriKn I think the problem is the mssql database which needs at least 2 GB of RAM to start. If you deploy this one somewhere else it could be usable on QNAP NAS (but it might not be your planned purpose to use something else than one device).

**Comment by cryptochrome** [comment]
How about providing a docker package for QNAP's Container Station, @kspearrin? It's natively Docker, but can't be controlled from the command line (well it can, but not supported), so your installation steps where you have to download and run some Bitwarden scripts to get the Docker machine up and running aren't supported. But I am sure it could still be installed with a few tweaks to the setup process.

**Comment by kspearrin** [comment]
Seems that to make that work you need a single Docker image with all services running on it.

**Comment by cryptochrome** [comment]
Correct. Everything would have to be in the container for it to properly work from within Container Station.

**Comment by kspearrin** [comment]
We are moving away from tracking Bitwarden feature requests like this one as GitHub issues. We have created a [Bitwarden Community Forum](https://community.bitwarden.com) which has a section for submitting, voting for, and discussing product [feature requests](https://community.bitwarden.com/c/feature-requests).

Please [sign up on our forums](https://community.bitwarden.com/signup), search to see if this request already exists. If so, you can vote for it and contribute to any discussions about it. If not, you can re-create the request there so that it can be properly tracked.

This issue will now be closed. Thanks!
